api/lib/api/endpoint.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Api.Endpoint do
+  use Phoenix.Endpoint, otp_app: :api
+
+  socket "/socket", Api.UserSocket
+
+  # Serve at "/" the static files from "priv/static" directory.
+  #
+  # You should set gzip to true if you are running phoenix.digest
+  # when deploying your static files in production.
+  plug Plug.Static,
+    at: "/", from: :api, gzip: false,
+    only: ~w(css fonts images js favicon.ico robots.txt)
+
+  # Code reloading can be explicitly enabled under the
+  # :code_reloader configuration of your endpoint.
+  if code_reloading? do
+    plug Phoenix.CodeReloader
+  end
+
+  plug Plug.RequestId
+  plug Plug.Logger
+
+  plug Corsica,
+    origins: "*",
```

Comment:
This would be unacceptable for production, but should be fine for our local development.  "http://localhost", "http://www.mysite.com"